docs/REST API/rest-api.md

```diff
@@ -4721,6 +4623,27 @@ To wipe a macOS, iOS, iPadOS, or Windows host, the host must have MDM turned on.
 }
 ```
 
+### Cancel host's upcoming activity
+
+`DELETE /api/v1/fleet/hosts/:id/activities/upcoming/:activity_id`
+
+#### Parameters
+
+| Name | Type    | In   | Description                  |
+| ---- | ------- | ---- | ---------------------------- |
+| id   | integer | path | **Required**. The host's ID. |
+| activity_id   | string | path | **Required**. The ID of host's upcoming activity. |
+| page | integer | query | Page number of the results to fetch.|
+| per_page | integer | query | Results per page.|
```

Comment:
Guessing these were copied/pasted by accident?
```suggestion
```

Reply:
Yes, it was copied by accident. Thanks for catching it!